src/data-sources/DataSourceConnector.ts

```diff
@@ -97,6 +97,25 @@ export class DataSourceConnector {
         return;
       }
 
+      // Checking if this is a config value
+      if (dependency.startsWith('connection:')) {
+        const connection = dependency.substr(dependency.indexOf(':') + 1);
+        const ds: IDataSource = this.dataSources[Object.keys(this.dataSources)[0]];
+        if ( !ds.plugin.hasOwnProperty('connections')) {
```

Comment:
^ seemed like all the datasources contain the connections config, so just picking the first one.